$OpenBSD: patch-src_include_common_Constants_h,v 1.1 2011/12/27 11:51:37 dcoppa Exp $

Fix bad #define (git commit b2ff838be24f5effae9ac4b75bd1897b710a60dc)

--- src/include/common/Constants.h.orig	Tue Dec 27 10:44:55 2011
+++ src/include/common/Constants.h	Tue Dec 27 10:45:22 2011
@@ -24,7 +24,7 @@
 //
 
 #ifndef COMMONCONSTANTS_H
-#define COMMONMACROS_H
+#define COMMONCONSTANTS_H
 
 const unsigned UNLIMITED = 0;
 
